### PR TITLE
light prefab port + map work

### PR DESCRIPTION
**Pull request by sousou63**
- Port of the light prefab
- Light prefab placement on the map
- Atm placements
- Fence placements

### CODE REVIEW

**Comment by jusvit** [comment]
Shouldn't those type of lights be baked into the map not placed with components for performance reasons.

File: Assets/prefabs/player/player.prefab [hunk 1]
```diff
@@ -95,7 +95,7 @@
         "Height": 64,
         "IgnoreLayers": "ragdoll,invis,pickup,no_player,fire",
         "Radius": 12,
-        "StepHeight": 42.65,
+        "StepHeight": 18,
```
Comment (jusvit) [inline Assets/prefabs/player/player.prefab:98]: Wasn't this just set to fix climbing stairs?

**Review by jusvit — CHANGES_REQUESTED** [review]

File: Assets/prefabs/player/player.prefab (hunk 1 above)
Comment (sousou63) [reply Assets/prefabs/player/player.prefab:98]: This value is the default one and it work the best, before it was set to something like 1.1

**Comment by sousou63** [comment]
No worries I've took the time to optimize them ( no impact on performances at all ) and are easy to place because of the prefab. This is not really a final solution more like a temporary solution to the map lightning meanwhile we get more comfortable with hammer. facepunch also want to get rid of hammer in the future to replace it with the scene so we will see.

**Review by jusvit — APPROVED** [review]

**Pull request merged**